lib/src/downstream/opentelemetry_downstream.rs

```diff
@@ -75,26 +77,22 @@ impl OpenTelemetryDownstream {
                                 metrics: batch,
                             }],
                         }],
-                    });
-                    match future.await {
-                        Ok(success) => {
-                            log::info!("sent metrics: {success:?}");
-                        }
-                        Err(err) => {
-                            if !err.metadata().is_empty() {
-                                log::error!(
-                                    "failed to send metrics: {err}. Metadata: {:?}",
-                                    err.metadata()
-                                );
-                            }
-                            log::error!("failed to send metrics: {err:?}")
+                    })
+                    .await;
+                match result {
+                    Ok(success) => {
+                        log::info!("sent metrics: {success:?}");
+                    }
+                    Err(err) => {
+                        if !err.metadata().is_empty() {
+                            log::error!(
+                                "failed to send metrics: {err}. Metadata: {:?}",
+                                err.metadata()
+                            );
                         }
-                    };
-                }
-                Err(timeout) => {
-                    log::debug!("no metrics activity: {timeout}");
-                    Delay::new(interval).await;
-                }
+                        log::error!("failed to send metrics: {err:?}")
```

Comment:
Same thing, should probably be debug since this means it was empty

Reply:
no, look at the loop definition - we only do work in here when there is a batch to send now.

Reply:
Ah gotcha, my GH display formatting was making it appear as if this was in a different portion of the loop